zeusd/src/main.rs

```diff
@@ -37,16 +38,23 @@ async fn main() -> anyhow::Result<()> {
             )?;
             tracing::info!("Listening on {}", &config.socket_path);
 
-            start_server_uds(listener, device_tasks, num_workers)?.await?;
+            start_server_uds(
+                listener,
+                device_tasks,
```

Comment:
I think all previous occurrences of `device_tasks` should be renamed to `gpu_device_tasks`.